src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -225,6 +225,54 @@ USER $_DEV_CONTAINERS_IMAGE_USER
 `;
 }
 
+function getFeatureInstallWrapperScriptTemplate() {
+    return `#!/bin/sh
+set -e
+
+on_exit () {
+	[ $? -eq 0 ] && exit
+	echo 'ERROR: Feature "#{name}" (#{id}) failed to install! Look at the documentation at #{documentation} for help troubleshooting this error.'
```

Comment:
If the feature doesn't have a `documentationUrl`, this sentence will read a little bit weird.

Reply:
Same with `name`, which this PR makes optional

Reply:
Agreed. I can tweak the error message in those cases.

Reply:
Updated `name` and `id` to default to "Unknown" if they are undefined to make it read a bit better. I also removed the second sentence from the message if no documentation url is defined.